indico/modules/events/editing/client/js/selectors.js

```diff
@@ -173,6 +173,25 @@ export const needsSubmitterConfirmation = createSelector(
 );
 export const getStaticData = state => state.staticData;
 
+export const getFileTypes = createSelector(
+  getStaticData,
+  staticData => {
+    return [...staticData.fileTypes].map(fileType => {
+      if (fileType.filenameTemplate) {
```

Comment:
```
      if (fileType.filenameTemplate) {
        return fileType;
      }
```

that way you can save some indentation since you don't need `else { ... }` around the rest (and generally "exit/return early" helps making functions more readable)